File: guanabara-java/src/com/danillo/algoritmos/lista01/Exercicio05.java
package com.danillo.algoritmos.lista01;

import java.util.Scanner;

public class Exercicio05 {
    public static void main(String[] args){
        Scanner entrada = new Scanner(System.in);

        System.out.println("Insira a nota 1: ");
        float n1 = entrada.nextFloat();
        System.out.println("Insira a nota 2: ");
        float n2 = entrada.nextFloat();

        float media = (n1 + n2)/2;
        System.out.println("A média entre " +n1+ " e " +n2+ " é igual a " +media);
    }
}

File: guanabara-c/lista01/exercicio05.c
#include <stdio.h>

int main(){
    float n1, n2, media;

    printf("Insira a primeira nota: ");
    scanf("%f", &n1);
    printf("Insira a segunda nota: ");
    scanf("%f", &n2);

    media = (n1 + n2)/2;
    printf("A média é: %.2f", media);

    return 0;
}
